index.js

```diff
@@ -46,6 +46,8 @@ function getPoolKey (externalSchemas, options) {
   const ajvConfig = JSON.stringify(options)
   return `${externals}${ajvConfig}`
 }
-module.exports = ValidatorSelector
+module.exports = AjvCompiler
+module.exports.default = AjvCompiler
+module.exports.AjvCompiler = AjvCompiler
```

Comment:
Should it be `fastifyAjvCompiler`?

Reply:
I did not name it like that, because we dont use fastify-plugin for this project.

But I can name it to that.

Reply:
I think the naming convention is fine even if it is a fastify server's option

Reply:
Erm... so rename to fastifyAjvCompiler?

Reply:
I would actually handle this like fast-uri and co and leave it as is.